README.md

```diff
@@ -126,6 +126,7 @@
 - [rlille.fr](https://github.com/RLille/rlille.fr) - The R Lille (R User Group) website using Quarto.
 - [R-Manuals](https://github.com/rstudio/r-manuals) - R Manuals rewritten with Quarto.
 - [Quarto tip a day](https://github.com/mine-cetinkaya-rundel/quarto-tip-a-day) - Website/blog highlighting a tip for Quarto every day.
+- [Quato Github Actions](https://github.com/jeremiahpslewis/quarto-reproducible-gh-actions) - Repository demonstrating how Github Actions can be used to automatically generate a Quato website
```

Comment:
How does it fit compared and/or in addition to the official repository https://github.com/mcanouil/awesome-quarto#continuous-integration--continuous-deployment (added in e995b9532a93275de2da19bb605164440e567f61)?